venues/naacl.org/NAACL/2018/Preprint/python/admin-init.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+
+import sys, os
+import argparse
+import openreview
+import config
+
+"""
+
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+groups = {}
```

Comment:
I don't see reason to use dictionary here, using an array is fine. You only need to iterate through the array to post each group